src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../types/react-tether/index.d.ts" /> Required to make dts-generator bundle react-tether definition file.
+
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  constantClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  display: string;
+  className: string;
+}
+
+export const userFeedbackState = {
+  VALID: 'VALID',
+  WARNING: 'WARNING',
+  ERROR: 'ERROR',
+};
+
+export class UserFeedback extends React.Component<IUserFeedbackProps, any> {
+
+  private getDefaultUserFeedbackStyle(): IUserFeedbackStyle {
+    return {
+      display: 'none',
```

Comment:
userfeedback 2 is display:block, userfeedback 3 is display:inline-block

it made sens to me because I was not using tether, and I did not because I do not think it is possible in the context I am using the UserFeedback